test/launch.js

```diff
@@ -103,4 +101,13 @@ describe("nearleyc", function() {
         var crontabResults = read('test/classic_crontab.results');
         parse(classicCrontab, crontabTest).should.deep.equal([JSON.parse(crontabResults)]);
     });
+
+    it('parentheses', function() {
+        // Try compiling the grammar
+        var parentheses = compile(read("examples/parentheses.ne"));
+        var parenthesesTest1 = '[(((<>)()({})())(()())(())[])]';
+        var parenthesesTest2 = '<<[([])]>([(<>[]{}{}<>())[{}[][]{}{}[]<>[]{}<>{}<>[]<>{}()][[][][]()()()]({})<[]>{(){}()<>}(<>[])]())({})>';
+        parse(parentheses, parenthesesTest1).should.deep.equal([true]);
+        parse(parentheses, parenthesesTest2).should.deep.equal([true]);
```

Comment:
Please kindly help me with Chai:

```

        var failCases = [
            ' ',
            '[}',
            '[(){}><]',
            '((((())))(())()'
        ];

        for (let i in failCases) {
            parse(parentheses, failCases[i]);
            (function() { parse(parentheses, failCases[i]); }).should.throw(Error);
        }
```

I will get this error:

```
  1) nearleyc parentheses:
     Error: nearley: No possible parsings (@0: ' ').
      at Parser.feed (lib/nearley.js:273:23)
      at parse (test/_shared.js:14:7)
      at Context.<anonymous> (test/launch.js:126:13)
```

If I remove the line `parse(parentheses, failCases[i]);`, I will get:

```
  1) nearleyc parentheses:
     AssertionError: expected [Function] to throw Error
      at Context.<anonymous> (test/launch.js:127:76)
```

That's really confusing. Clearly an instance of Error is being thrown. Thanks.